plugins/inputs/directory_monitor/README.md

```diff
@@ -0,0 +1,59 @@
+# Directory Monitor Input Plugin
+
+This plugin monitors a single directory (without looking at sub-directories), and takes in each file placed in the directory.
+The plugin will gather all files in the directory at a configurable interval (`monitor_interval`), and parse the ones that haven't been picked up yet.
+
+Please be advised that this plugin pulls files directly after they've been in the directory for the length of the configurable `directory_duration_threshold`, and thus files should not be written 'live' to the monitored directory unless they are guaranteed to finish writing before the `directory_duration_threshold`. This plugin is intended to read files that are moved or copied to the monitored directory, and thus files should also not be used by another process or else they may fail to be gathered.
```

Comment:
Seems like this approach is asking for trouble. It's friendlier to the users to allow them to write directly into the directory, but if they're wrong about how long the write takes they'll lose data. At the least I think we should encourage them to write the files elsewhere then move them into the folder when complete.